scripts/install-horovod-for-rtd.sh

```diff
@@ -0,0 +1,18 @@
+# This script is for installing horovod on readthedocs only!
```

Comment:
Do we still need this file? @lgarithm

Reply:
This file was renamed to [install-requirements-for-rtd.sh](https://github.com/tensorlayer/tensorlayer/blob/master/scripts/install-requirements-for-rtd.sh), it is used in [setup.py](https://github.com/tensorlayer/tensorlayer/blob/master/setup.py#L99)

Reply:
@zsdonghao please remove this file then.